packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/tags/GetTagsTests.java

```diff
@@ -226,4 +236,93 @@ public void checkThatDeletedTagIsNotRetrievedAnymore() throws Exception
         returnedCollection.assertThat().entriesListIsNotEmpty()
                 .and().entriesListDoesNotContain("tag", removedTag.toLowerCase());
     }
+
+    /**
+     * Verify if count field is present and with value 0 for searched tag.
+     */
+
+    @Test(groups = { TestGroup.REST_API, TestGroup.TAGS, TestGroup.REGRESSION })
+    public void testGetTag_includingCount()
+    {
+        STEP("Create single tag as admin including count and verify if it is 0");
+        final RestTagModel tagModel = createTagModelWithName(getRandomName(TAG_NAME_PREFIX).toLowerCase());
+        final RestTagModel createdTag = restClient.authenticateUser(adminUserModel).withCoreAPI().include(FIELD_COUNT).createSingleTag(tagModel);
+
+        restClient.assertStatusCodeIs(CREATED);
+
+        STEP("Get a single tag, including count and verify if it is 0");
+        final RestTagModel searchedTag = restClient.withCoreAPI().include(FIELD_COUNT).getTag(createdTag);
+        restClient.assertStatusCodeIs(OK);
+        searchedTag.assertThat().field(FIELD_TAG).is(tagModel.getTag())
+                .assertThat().field(FIELD_ID).isNotEmpty()
+                .assertThat().field(FIELD_COUNT).is(0);
+    }
+
+    /**
+     * Verify if count field is not present for searched tag.
+     */
+
+    @Test(groups = { TestGroup.REST_API, TestGroup.TAGS, TestGroup.REGRESSION })
+    public void testGetTag_notIncludingCount()
+    {
+        STEP("Create single tag as admin including count and verify if it is 0");
+        final RestTagModel tagModel = createTagModelWithName(getRandomName(TAG_NAME_PREFIX).toLowerCase());
+        final RestTagModel createdTag = restClient.authenticateUser(adminUserModel).withCoreAPI().include(FIELD_COUNT).createSingleTag(tagModel);
+
+        restClient.assertStatusCodeIs(CREATED);
+
+        STEP("Get a single tag, not including count and verify if it is not present in the response");
+        final RestTagModel searchedTag = restClient.withCoreAPI().getTag(createdTag);
+        restClient.assertStatusCodeIs(OK);
+        searchedTag.assertThat().field(FIELD_TAG).is(tagModel.getTag())
+                .assertThat().field(FIELD_ID).isNotEmpty()
+                .assertThat().field(FIELD_COUNT).isNull();
+    }
+
+    /**
+     * Verify if count field is present for searched tags.
+     */
+
+    @Test(groups = { TestGroup.REST_API, TestGroup.TAGS, TestGroup.REGRESSION })
+    public void testGetTags_includingCount()
+    {
+        STEP("Create single tag as admin including count and verify if it is 0");
```

Comment:
I don't think we need to create a tag for this test, because you don't care whether it's in use or not.

Reply:
Same comment for the next test too.

Reply:
I care about many things but here you are right, thanks